src/classes/GE_PaymentServices.cls

```diff
@@ -144,14 +154,20 @@ public with sharing class GE_PaymentServices {
         return config.get(PS_IntegrationServiceConfig.SFDO_GATEWAYIDS);
     }
 
+    public String getMerchantIds() {
+        return config.get(PS_IntegrationServiceConfig.SFDO_MERCHANTIDS);
+    }
+
     /*******************************************************************************************************
     * @description Inner class to assist in constructing the Payload portion of a JSON Web Token.
     */
     public with sharing class JWTPayload {
         String sfdoId;
+        String role;
 
-        public JWTPayload(String sfdoId) {
+        public JWTPayload(String sfdoId, String role) {
```

Comment:
Should we pluralize `role`? I know it's technically a single string, but multiple roles comma-separated can be contained within this string right?

Reply:
@psadie-sfdo `JWTPayload` isn't being used anymore so we don't have to worry about this one.